napari_allencell_annotator/assets/test.csv

```diff
@@ -0,0 +1,3 @@
+Shuffled:,FALSE,,,,
+Annotations:,"{""name"": {}, ""name2"": {}, ""name3"": {}}",,,,
```

Comment:
if these are for testing can you put them into _tests/assets
if not just remove them